dashboard/config/routes.rb

```diff
@@ -466,6 +466,9 @@
             get 'page/:puzzle_page', to: 'script_levels#show', as: 'puzzle_page', format: false
             # /s/xxx/lessons/yyy/levels/zzz/sublevel/sss
             get 'sublevel/:sublevel_position', to: 'script_levels#show', as: 'sublevel', format: false
+            # Get the level data via JSON.
+            # /s/xxx/lessons/yyy/levels/zzz/level_data
+            get 'level_data', to: 'script_levels#level_data'
```

Comment:
Oh, good point.  For a lot of this work, I'm still using the existing plumbing, and then implementing specialized behavior to enable switching-with-reloads for a script-level in particular.  I wonder whether it would be alright to use the existing code for a `/levels/:level_id` page for now, to keep this change scoped, and since it's already working, and we can potentially migrate it later?

Reply:
Good call.  I've added the equivalent route and made it work on a level page.